sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.applicationChangeRequest.e2e-spec.ts

```diff
@@ -39,20 +43,27 @@ describe("ApplicationStudentsController(e2e)-applicationChangeRequest", () => {
   let db: E2EDataSources;
   let programYear: ProgramYear;
   let defaultPayload: SaveApplicationAPIInDTO;
+  let dynamicFormConfigurationService: DynamicFormConfigurationService;
 
   beforeAll(async () => {
     const { nestApplication, module, dataSource } =
       await createTestingAppModule();
     app = nestApplication;
     appModule = module;
     db = createE2EDataSources(dataSource);
+    dynamicFormConfigurationService = app.get(DynamicFormConfigurationService);
     // Program Year for the following tests.
     const formService = await getProviderInstanceForModule(
       appModule,
       AppStudentsModule,
       FormService,
     );
     programYear = await ensureProgramYearExists(db, 2050);
+    // Ensure the dynamic form configuration for part time exists.
+    await createPYStudentApplicationFormConfiguration(
```

Comment:
This method has the potential to be useful many times, and a factory would be nice. Is there a plan to do it in the upcoming E2E tests?

Reply:
I can move this method to a factory in current PR itself.